README.md

```diff
@@ -178,15 +182,12 @@ version conflicts that it cannot resolve. The two Guava targets can then be used
 in BUILD files like so:
 
 ```python
-load("@rules_jvm_external//:defs.bzl", "artifact")
```

Comment:
piggybacking this PR to remove these artifact() references too

Reply:
ok